<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>enrich_go — enrich_kegg • RNAseqStat</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="enrich_go — enrich_kegg"><meta property="og:description" content="a funtion group deg data frame and make enrichment analysis in one step"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=G-PMYT07QDM3"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-PMYT07QDM3');
</script></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">RNAseqStat</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../index.html">
    <span class="fa fa-home"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">
    <span class="fa fa-code"></span>
     
    Functions and data
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="paper paper-plane"></span>
     
    Guide
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li class="dropdown-header">Workflow</li>
    <li>
      <a href="../articles/Check_your_data.html">1. Check your data</a>
    </li>
    <li>
      <a href="../articles/DEG_analysis.html">2. DEG analysis</a>
    </li>
    <li>
      <a href="../articles/go.html">3. GO</a>
    </li>
    <li>
      <a href="../articles/kegg.html">4. KEGG</a>
    </li>
    <li>
      <a href="../articles/gseKegg.html">5. gseKEGG</a>
    </li>
    <li class="divider">
    <li class="dropdown-header">Visualizations</li>
    <li>
      <a href="../articles/visualization_deg_volcano.html">Visualization DEG Volcano</a>
    </li>
    <li>
      <a href="../articles/visualization_deg_heatmap.html">Visualization DEG Heatmap</a>
    </li>
    <li>
      <a href="../articles/visualization_go_barplot.html">Visualization GO Barplot</a>
    </li>
    <li>
      <a href="../articles/visualization_kegg_barplot.html">Visualization KEGG Barplot</a>
    </li>
    <li>
      <a href="../articles/visualization_gsekegg_barplot.html">Visualization gseKEGG Barplot</a>
    </li>
    <li>
      <a href="../articles/visualization_gseplot.html">Visualization GSE plot</a>
    </li>
    <li class="divider">
    <li class="dropdown-header">Tools</li>
    <li>
      <a href="../articles/tools_groupDEG.html">Group DEG</a>
    </li>
    <li>
      <a href="../articles/tools_topDEG.html">Choose top DEG</a>
    </li>
  </ul></li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/xiayh17/RNAseqStat/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>enrich_go</h1>
    <small class="dont-index">Source: <a href="https://github.com/xiayh17/RNAseqStat/blob/HEAD/R/erich_kegg.R" class="external-link"><code>R/erich_kegg.R</code></a></small>
    <div class="hidden name"><code>enrich_kegg.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>a funtion group deg data frame and make enrichment analysis in one step</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="fu">enrich_kegg</span><span class="op">(</span>
  <span class="va">deg_data</span>,
  <span class="va">x</span>,
  <span class="va">y</span>,
  cut_FC <span class="op">=</span> <span class="fl">1</span>,
  cut_P <span class="op">=</span> <span class="fl">0.05</span>,
  organism <span class="op">=</span> <span class="st">"hsa"</span>,
  keyType <span class="op">=</span> <span class="st">"kegg"</span>,
  OrgDb <span class="op">=</span> <span class="st">"org.Hs.eg.db"</span>,
  pvalueCutoff <span class="op">=</span> <span class="fl">0.05</span>,
  pAdjustMethod <span class="op">=</span> <span class="st">"BH"</span>,
  minGSSize <span class="op">=</span> <span class="fl">10</span>,
  maxGSSize <span class="op">=</span> <span class="fl">500</span>,
  qvalueCutoff <span class="op">=</span> <span class="fl">0.2</span>,
  use_internal_data <span class="op">=</span> <span class="cn">FALSE</span>,
  label <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Down"</span>, <span class="st">"Stable"</span>, <span class="st">"Up"</span><span class="op">)</span>,
  label_ns <span class="op">=</span> <span class="st">"Stable"</span>,
  mc.cores <span class="op">=</span> <span class="fl">1L</span>
<span class="op">)</span></code></pre></div>
    </div>

    <div id="arguments">
    <h2>Arguments</h2>
    <dl><dt>deg_data</dt>
<dd><p>a DEG data frame contains logFC and p value</p></dd>


<dt>x</dt>
<dd><p>which column is log FC</p></dd>


<dt>y</dt>
<dd><p>which column is P value</p></dd>


<dt>cut_FC</dt>
<dd><p>a single number character or numeric vector in threshold value of log FC</p></dd>


<dt>cut_P</dt>
<dd><p>threshold value of P value, can set for every cut_FC number in numeric vector format</p></dd>


<dt>organism</dt>
<dd><p>supported organism listed in 'http://www.genome.jp/kegg/catalog/org_list.html'</p></dd>


<dt>keyType</dt>
<dd><p>one of "kegg", 'ncbi-geneid', 'ncib-proteinid' and 'uniprot'</p></dd>


<dt>OrgDb</dt>
<dd><p>OrgDb</p></dd>


<dt>pvalueCutoff</dt>
<dd><p>adjusted pvalue cutoff on enrichment tests to report</p></dd>


<dt>pAdjustMethod</dt>
<dd><p>one of "holm", "hochberg", "hommel", "bonferroni", "BH", "BY", "fdr", "none"</p></dd>


<dt>minGSSize</dt>
<dd><p>minimal size of genes annotated by Ontology term for testing.</p></dd>


<dt>maxGSSize</dt>
<dd><p>maximal size of genes annotated for testing</p></dd>


<dt>qvalueCutoff</dt>
<dd><p>qvalue cutoff on enrichment tests to report as significant. Tests must pass i) pvalueCutoff on unadjusted pvalues, ii) pvalueCutoff on adjusted pvalues and iii) qvalueCutoff on qvalues to be reported.</p></dd>


<dt>use_internal_data</dt>
<dd><p>logical, use KEGG.db or latest online KEGG data</p></dd>


<dt>label</dt>
<dd><p>symbol word for groups</p></dd>


<dt>label_ns</dt>
<dd><p>which group is the stable group</p></dd>


<dt>mc.cores</dt>
<dd><p>param for mclapply, choose cores to use in non-Windows machine</p></dd>

</dl></div>
    <div id="value">
    <h2>Value</h2>
    

<p>enrichKEGG result</p>
    </div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span class="kw">if</span> <span class="op">(</span><span class="cn">FALSE</span><span class="op">)</span> <span class="op">{</span></span>
<span class="r-in"><span class="fu">enrich_kegg</span><span class="op">(</span>deg_data <span class="op">=</span> <span class="va">DEG_df</span>, x <span class="op">=</span> <span class="st">"log2FoldChange"</span>, y <span class="op">=</span> <span class="st">"pvalue"</span><span class="op">)</span></span>
<span class="r-in"><span class="op">}</span></span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by <a href="https://github.com/jmzeng1314" class="external-link">Jianming Zeng</a>, <a href="https://github.com/xiayh17" class="external-link">Yonghe Xia</a>, <a href="https://space.bilibili.com/338686099" class="external-link">Biotrainee group</a>.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.4.</p>
</div>

      </footer></div>

  


  

  </body></html>

